src/commands/status/core.js

```diff
@@ -0,0 +1,114 @@
+const { table } = require('table');
+const fetch = require('node-fetch');
+
+const BaseCommand = require('../../oclif/command/BaseCommand');
+const CoreService = require('../../core/CoreService');
+
+const ContainerIsNotPresentError = require('../../docker/errors/ContainerIsNotPresentError');
+
+class CoreStatusCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @param {CoreService} coreService
+   * @param {Config} config
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    args,
+    flags,
+    dockerCompose,
+    createRpcClient,
+    config,
+  ) {
+    const rows = [];
+
+    const coreService = new CoreService(
+      createRpcClient(
+        {
+          port: config.get('core.rpc.port'),
+          user: config.get('core.rpc.user'),
+          pass: config.get('core.rpc.password'),
+        },
+      ),
+      dockerCompose.docker.getContainer('core'),
+    );
+
+    let insightUrl;
+    switch (config.options.network) {
```

Comment:
I would recommend to define an object with key as network name and value as insight URL so you can do just `insightURLs[networkName]` to get specific URL. It will look cleaner than switch.

Reply:
Thanks, these little style hints are incredibly useful.